pc/filter/voxelgrid/option.go

```diff
@@ -0,0 +1,18 @@
+package voxelgrid
+
+import (
+	"github.com/seqsense/pcgol/mat"
+)
+
+type Options struct {
+	LeafSize  mat.Vec3
+	ChunkSize [3]int
```

Comment:
`ChunkSize` is the number of parts we want the pointcloud to be split into in each direction not a distance. Is that correct?